### ISSUE TITLE
USER STORY: User Profile

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/Sleep-Healthily#26]
### If applicable, add a related Epic.

Epic: #24 

### User Story

As a **user**, I can **access and view my user profile** so that I can **see my personal information, order history, and manage my account settings**.

### Acceptance Criteria

- The user should be able to easily navigate to their profile from any page.
- The profile should display relevant personal information and order summary.
- Users should be able to update their information and settings from their profile.

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Updated the issue with any relevant notes or feedback (to date).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Design and implement the user profile page using HTML, CSS, JS and Bootstrap.
- [x] Develop backend functionality in Django to display and update user information.
- [x] Ensure the profile page is responsive and user-friendly.

**Comment by stephendawsondev** [comment]
This may need to be put back into the backlog just because I will need to complete the order summary before I can really mark this off.

**Comment by stephendawsondev** [comment]
This will be put on hold until #12 is completed, because that also needs to be added to the profile page.
